fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -223,7 +223,7 @@
 
     // Enable use of legacy whole-system search parameters for the search request
     private final boolean legacyWholeSystemSearchParamsEnabled;
-    
+
```

Comment:
Whitespace cleanup may cause merge issues with currently active PRs

Reply:
Eclipse automatically cleans up the whitespace, so it was either do it now or later.  I thought now would be fine.